book/website/collaboration/inclusive-events/inclusive-events-location.md

```diff
@@ -0,0 +1,249 @@
+
+If your event uses buffet-style food, options should be clearly labelled with dietary information and potential allergens. 
+Try to include the ingredients list on table cards to help attendees with specific dietary concerns identify suitable options more easily. 
+
+If your meal includes a "sit down" or formal service, try to prioritize the serving of dietary requirement, or accommodated meals first. 
+This can allow for any additional time that may be needed to fix a catering error and keep attendees with severe allergies safe. 
+Try to do this in a way that is not timed so separately from the rest of the meals that accommodated meals are finished or sitting with cold food while waiting for the remaining meals to be served. 
+By working to avoid segregating meals in time, it ensures that everyone can enjoy their meals simultaneously and feel equally able to engage in conversation over meals. 
+
+You should also consider the needs of attendees with specific mobility or sight impairments during breaks and mealtimes, especially if the food is buffet-style. 
+Providing support, such as assistance in carrying plates of food and drinks, ensures that these individuals can navigate the catering area comfortably. 
+
+If your event is hosted in person but does not provide meals or snacks, consider creating a list of nearby restaurants that offer a variety of dietary options, 
+including vegetarian, vegan, kosher, and gluten-free food. This can be helpful in supporting attendees unfamiliar with the area to locate food suitable for them. 
+
+Lastly, for events coinciding with Ramadan, you should consider making arrangements to accommodate attendees who are fasting. 
+For example, by extending available catering times so that food can be prepared and accessed after sunset and before dawn. 
+You may also consider splitting or adjusting meal times so that groups of attendees have the option of eating meals together. 
+For example by providing earlier meals to attendees who require it such as those who experience fatigue or have caring duties for children with an early bedtime, and then a later meal after sunset for those who wish to stay up. 
+
+As with many of the examples suggested in the chapter, it can be difficult to find a "perfect answer" that caters to the specific needs of all attendees. 
+The most important step is to identify how you will work to be inclusive of your attendees. 
+In this example, it may not be possible to have all attendees eat at the same time.
+As organizers, you may decide that offering separate meal times to ensure that attendees are catered for is more important than having everyone in the same room at the same time. 
+It is incredibly beneficial to work with a representative with lived experience to strategize solutions in cases where requirements may seem to conflict. 
+
+(cl-inclusive-events-location-venue)=
+## Venue Planning
```

Comment:
I think you already did mention it briefly in the "picking a location" bit, but I think it's worth to explicitly reference this section in the top section as "see below for specific things to also think about when doing a tour of potential locations"